mindesa_purchase/models/purchase_order.py

```diff
@@ -31,17 +28,16 @@ def confirm_rfq_action(self, automatic=True):
             cr.commit()
```

Comment:
remove all checks on line 18 after filters and domains changed in the cron itself